gpu4pyscf/drivers/dft_driver.py

```diff
@@ -133,11 +135,17 @@ def run_dft(mol_name, config, charge=None, spin=0):
         mf = mf.to_gpu()
 
     mf.chkfile = None
-    if with_solvent:
+    if with_solvent and config['solvent']['method'].endswith(('PCM', 'pcm')):
         mf = mf.PCM()
         mf.with_solvent.lebedev_order = 29
         mf.with_solvent.method = config['solvent']['method'].replace('PCM','-PCM')
         mf.with_solvent.eps = config['solvent']['eps']
+    
+    if with_solvent and config['solvent']['method'].endswith(('smd', 'SMD')):
+        mf = mf.SMD()
+        mf.with_solvent.lebedev_order = 29
+        mf.with_solvent.method = 'SMD'
+        mf.with_solvent.solvent = config['solvent']['solvent']
```

Comment:
['solvent'] might not be specified. `with_solvent.solvent` can be the default value.